blocklist-client/src/api/routes.rs

```diff
@@ -0,0 +1,17 @@
+use super::handlers;
+use crate::config::Settings;
+use std::convert::Infallible;
+use warp::Filter;
+
+pub fn routes(
+    settings: &Settings,
+) -> impl Filter<Extract = impl warp::Reply, Error = Infallible> + Clone {
+    let config = settings.risk_analysis.clone();
+
+    warp::path("screen")
+        .and(warp::path::param::<String>())
+        .and(warp::get())
+        .and(warp::any().map(move || config.clone()))
```

Comment:
While passing the config around this way is an okay solution, I expect the configuration to be statically known through environment variables. So I would opt for having a static read-only config in `config.rs` which is lazily instantiated from environment variables using `once_cell::sync::Lazy` [link](https://crates.io/crates/once_cell).

If there are a lot of env variables to be set, we can use `dotenv` [link](https://crates.io/crates/dotenv) to load them on startup from a file.

I'm happy to have a conversation on this. I know we haven't really talked about configuration management so far. My take is that, since we are containerizing the applications we should follow the twelve-factor-app philosophy which dictates the use of env variables here. https://12factor.net/config

Reply:
If it's unclear what I'm referring to here, feel free to leave it as-is and I can create a follow-up PR to change the configuration management.

Reply:
sure, we can go with the env var option, especially since for this client, I don't believe we'll have a lot of configs (not sure about the preference for the signer). 
But I will make the change in another PR if that's ok.